src/app/keyboard-demo/keyboard-demo/keyboard-demo.component.ts

```diff
@@ -15,6 +15,7 @@ export class KeyboardDemoComponent {
   }
 
   onTextChange(text: string) {
+    // eslint-disable-next-line no-console
     console.log(text);
```

Comment:
how about changing it to console.info? since you really want to keep it in this demo app i think that would make sense.